source/ShopPanel.cpp

```diff
@@ -353,7 +353,8 @@ void ShopPanel::DrawButtons()
 	
 	Point buyCenter = Screen::BottomRight() - Point(210, 25);
 	FillShader::Fill(buyCenter, Point(60, 30), back);
-	string BUY = (playerShip && selectedOutfit && player.Cargo().Get(selectedOutfit)) ? "_Install" : "_Buy";
+	string BUY = ((playerShip && selectedOutfit && player.Cargo().Get(selectedOutfit))
+		|| (player.Storage() && player.Storage()->Get(selectedOutfit))) ? "_Install" : "_Buy";
```

Comment:
Actually, I just fiddled around with this some more and if I press i it WILL install from storage to cargo, so it's just a matter of the button not working.

Reply:
I just made a change to make the button give the correct key-code.

Reply:
Thanks for the extensive testing. I went with the suggestion to label the button "Cargo". I tried "to Cargo" first, but either the text was too big or the button too small.

Reply:
The 'c' key now also works the same as the 'i' key. (So the cargo label also has a matching key.)